core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooCache.java

```diff
@@ -157,6 +158,20 @@ public void process(WatchedEvent event) {
       switch (event.getType()) {
         case NodeDataChanged:
         case NodeChildrenChanged:
+          if (event.getPath().endsWith("conf")
```

Comment:
is checking endsWith("conf") specific enough for table conf directories?  Is there any other path that ends with conf?